lib/src/query.rs

```diff
@@ -39,6 +39,10 @@ impl Query {
         self
     }
 
+    pub fn has_param_key(&self, key: &str) -> bool {
+        self.params.value.contains_key(&key.into())
```

Comment:
We just merged #96 and this PR added an `impl Borrow<str> for BoltString`, so you can use

```suggestion
        self.params.value.contains_key(key)
```

and avoid having to copy the key.